frontend/src/components/form/base/EParseField.vue

```diff
@@ -26,13 +26,22 @@ Displays a field as a textfield (can be used with v-model)
       type="text"
       v-on="inputListeners"
     >
-      <!-- passing through all slots -->
-      <slot v-for="(_, name) in $slots" :slot="name" :name="name" />
-      <slot name="scoped" :scoped-slots="$scopedSlots">
-        <template v-for="(_, name) in $scopedSlots" :slot="name" slot-scope="slotData">
-          <slot :name="name" v-bind="slotData" />
-        </template>
-      </slot>
+      <template #prepend>
```

Comment:
Why this change?

Reply:
Because it was not working correctly. I was not able to bind additional arguments to the slot, as it was the exact scoped slot of the v-text-field. I think it is sensible to not loop over the scoped slots anyway for vue3, as it works quite different there.